### PR TITLE
Create palindrome_string.c 

### LINKED ISSUE #2507: Palindrome string in C
A code to check whether the string is palindrome or not.

### PR DESCRIPTION
**Pull request by anshuman1510**
Fixes #2507 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [ ] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- 
- 

#### Languages Used:
- 
- 

#### Files Added:
- 
- 

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
please include the issue number

**Comment by anshuman1510** [comment]
Done.

File: Palindrome/palindrome_string.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <string.h> /*used string library header file for string functions.*/
+main()
```
Comment (SakshiShreya) [inline Palindrome/palindrome_string.c]: Please a space after line 2.

File: Palindrome/palindrome_string.c [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+{
+	char str[200], rev[200];
```
Comment (SakshiShreya) [inline Palindrome/palindrome_string.c]: Use 4 space indentation instead of 1 tab indentation.

File: Palindrome/palindrome_string.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+	printf ("Enter a string : ");
+	gets (str); /*This function is used to read multiword string (one line) from keyboard*/
+	strcpy (rev,str); /*This function is used to copy one string into another*/
+	strrev (rev); /*This function is used to reverse the giveen string*/
+	if (strcmp(str,rev)==0) /*To compare two strings.*/
+	printf ("\nThe Given string is Palindrome");
```
Comment (SakshiShreya) [inline Palindrome/palindrome_string.c]: Fix indentation in both if and else block.

File: Palindrome/palindrome_string.c [hunk 4]
```diff
@@ -0,0 +1,25 @@
+#include <stdio.h>
+#include <string.h> /*used string library header file for string functions.*/
+main()
+{
+	char str[200], rev[200];
+	printf ("Enter a string : ");
+	gets (str); /*This function is used to read multiword string (one line) from keyboard*/
+	strcpy (rev,str); /*This function is used to copy one string into another*/
```
Comment (SakshiShreya) [inline Palindrome/palindrome_string.c]: Add space after comma and around operators

File: Palindrome/palindrome_string.c (hunk 1 above)
Comment (anshuman1510) [reply Palindrome/palindrome_string.c]: Done

File: Palindrome/palindrome_string.c (hunk 2 above)
Comment (anshuman1510) [reply Palindrome/palindrome_string.c]: Done

File: Palindrome/palindrome_string.c (hunk 3 above)
Comment (anshuman1510) [reply Palindrome/palindrome_string.c]: Done

File: Palindrome/palindrome_string.c (hunk 4 above)
Comment (anshuman1510) [reply Palindrome/palindrome_string.c]: I have added space after comma operators.
Can you please tell what are these around operators.

**Comment by singh-shreya6** [comment]
Please squash your commits. The request has been made 17 days ago @anshuman1510. Else we will have to close PR due to inactivity.

**Pull request closed without merging**